src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -703,6 +703,17 @@ def CancelPathfinding() -> None:
     """
     pass
 
+def GetPath(x: int, y: int, z: int = 1337, distance: int = 1) -> List[Tuple[int, int, int]]:
```

Ruff (0.12.2)

Undefined name `List`

(F821)